src/components/Nav.js

```diff
@@ -11,11 +11,11 @@ const Nav = (props) => {
         })
     }, []);
     return (
-        <div class="nav">
+        <div className="nav">
             <Link to="/">all</Link>
-            {cats.map((item) => (
-                <Link to={`/reviews/${item.slug}`}>
-                <p class="nav-item">{item.slug}</p>
+            {cats.map((item, index) => (
+                <Link key={index} to={`/reviews/${item.slug}`}>
```

Comment:
Please use something else instead of the index for your key here! While it will technically work - it is considered bad practice, as the index refers to the position of the element, rather than its contents. If one element were deleted, all of the indexes would change. Like I say, this won't impact what we're working on currently, but would have ramifications as your project scaled-up in terms of usage.